packages/office-ui-fabric-react/src/components/Panel/Panel.scss

```diff
@@ -289,7 +289,7 @@
     right: 0;
     padding: 0 16px 20px;
     overflow-y: auto;
-    .ms-Panel--hasCloseButton & {
+    :global(.ms-Panel--hasCloseButton) & {
```

Comment:
Yeah, you fixed it in 1134. This should correct the problem until css modules is released.